### PR TITLE
Add largest characteristic speed to FFE system 

### PR DESCRIPTION
**Pull request by yoonso0-0**
## Proposed changes

Dependent on (first two commits) : 
- [ ] #4344 

### Code review checklist

- [ ] The code is documented and the documentation renders correctly. Run
  `make doc` to generate the documentation locally into `BUILD_DIR/docs/html`.
  Then open `index.html`.
- [ ] The code follows the stylistic and code quality guidelines listed in the
  [code review guide](https://spectre-code.org/code_review_guide.html).
- [ ] The PR lists upgrade instructions and is labeled `bugfix` or
  `new feature` if appropriate.

### CODE REVIEW

File: src/Evolution/Systems/ForceFree/Characteristics.hpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+namespace ForceFree {
+namespace Tags {
+/*!
+ * \brief Compute the largest characteristic speed of the GRFFE system with
+ * divergence cleaning.
+ *
+ * Wave speeds of the fast modes of the GRFFE system are the speed of light.
+ *
+ * \f{align*}
+ *  \lambda_\pm = - n_i \beta^i \pm \alpha
```
Comment (nilsdeppe) [inline src/Evolution/Systems/ForceFree/Characteristics.hpp]: This isn't the largest char speed. We compute the char speeds in the volume, so there's no face normal. See how GH does it.

**Review by nilsdeppe — CHANGES_REQUESTED** [review]

File: src/Evolution/Systems/ForceFree/Characteristics.hpp (hunk 1 above)
Comment (ermost) [reply src/Evolution/Systems/ForceFree/Characteristics.hpp]: Ok, so this is just for computing the global maximum speed used for defining the time step? -> Ok, I found the Cfl.hpp file :)

Comment (nilsdeppe) [reply src/Evolution/Systems/ForceFree/Characteristics.hpp]: Yes, that's right :)

**Comment by yoonso0-0** [comment]
Pushed a fixed one. Please let me know anything to correct further :)

**Review by nilsdeppe — COMMENTED** [review]
LGTM

**Comment by nilsdeppe** [comment]
Please ping me once you've rebased and we can merge this :)

**Comment by yoonso0-0** [comment]
@nilsdeppe Rebased on develop  :)

**Review by nilsdeppe — DISMISSED** [review]

**Comment by nilsdeppe** [comment]
CI failures are GH DuDt

**Pull request merged**